spec/importer/mahonia_mapper_spec.rb

```diff
@@ -44,6 +44,36 @@
     end
   end
 
+  it "provides an embargo date field as a Mahonia::EdtfLiteral" do
+    mapper.metadata = { "embargo_date" => ["1997-02-01 00:00"] }
+
+    expect(mapper.embargo_release_date.first).to be_an_instance_of(Mahonia::EdtfLiteral)
+  end
+
+  it "provides an embargo date field with a valid value" do
+    mapper.metadata = { "embargo_date" => ["1997-02-01 00:00"] }
+
+    expect(mapper.embargo_release_date.first.value).to eq("[\"1997-02-01 00:00\"]")
```

Comment:
@no-reply are these date values formatted correctly?

Reply:
I think what we want is to cast to `Date`, which may work simply with these dates as is, or we may need to strip the timestamp off.